Add onboarding journey to product page

tomskerous opened the pull request:
- Add on boarding page
- Add some links out to relevant bits of docs
- Add some inbound links
- Change roadmap page to direct “what it does now” users straight to
features, not via homepage.

tomskerous committed:
Add onboarding journey to product page

- Add on boarding page
- Add some links out to relevant bits of docs
- Add some inbound links
- Change roadmap page to direct “what it does now” users straight to
features, not via homepage.

tomskerous committed:
Anchors, Finance, Cruft

- Left some bits from the support page originally copied on the bottom
of this list.
- Added internal links.
- Added a mention of finance, following meeting with Clare Moss.

paroxp commented on source/roadmap.html.erb:
```diff
@@ -39,7 +39,7 @@ title: Roadmap
       <p>If you would like to find out more about these plans, or you have feedback on whether or not these features are useful for you, please <a href="/support.html">get in touch</a>. We are also keen for private beta partners on forthcoming features.</p>
 
-      <p>To see what GOV.UK PaaS offers right now, visit <a href="/">our product page</a>.</p>
+      <p>You can also <a href="/features.html">what GOV.UK PaaS offers right now</a>, or walk through a full <a href="/onboarding.html">onboarding and development journey</a>.</p>
```
This would output:

> You can also what GOV.UK PaaS offers right now, or walk through a full onboarding and development journey.

tomskerous replied:
good spot. Fixing this.

paroxp commented on source/onboarding.html.erb:
```diff
@@ -0,0 +1,124 @@
+---
+title: Support
```
I don't think, this should say `Support`.

tomskerous replied:
Fixed.

paroxp reviewed (changes requested):
Left some comments to issues.

Also, I'm not really sure about the long list of points. It's monotonic. IMO an individual will get bored with it and dismiss these steps, which then would mean that list is somewhat pointless.

I'm wondering if we should have an UX colleague look at this and advise?

tomskerous committed:
Fixing title and accidentally deleted word

- changed title of “Onboarding”
- put back in the word “see” that got caught in the crossfire after
some drastic surgery on links.

tomskerous commented:
Fixed the genuine bugs. Re length of list, perfect is the enemy of done here. Let's get it live and improve when we have capacity from UX and user feedback.
